src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkTask.java

```diff
@@ -68,6 +68,7 @@ public void put(Collection<SinkRecord> records) {
     try {
       writer.write(records);
     } catch (SQLException sqle) {
+      log.warn(sqle.getNextException().getMessage());
       log.warn(
           "Write of {} records failed, remainingRetries={}",
           records.size(),
```

Comment:
Why output this before the log message for the original? Also, it's possible that the next exception also has a next exception, so it'd be better to log all of them in the correct order _after_ the existing log message:

```
for (Throwable next : sqle) {
    log.warn(next);
}
```

WDYT?

Reply:
Very good point. Just updated.

Reply:
Just updated, please take another look. Thank you.